ubuntu_16.04.sh

```diff
@@ -20,17 +20,17 @@ export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
 export LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
 echo "export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu" >> ~/.bashrc
 cd $HOME
-mkdir opt
+mkdir -p opt
 cd opt
 # Install MOAB
-mkdir moab
+mkdir -p moab
 cd moab
 git clone https://bitbucket.org/fathomteam/moab
```

Comment:
We briefly discussed what to do about the error if someone tries to clone into a directory that already exists....

As is, this will produce an error.  I see two options:
1. we can force the deletion of any existing directory (possibly a subdirectory of this `moab` directory)
2. we can let the error stand and have the user correct it as they wish